astropy/stats/src/fast_sigma_clip.c

```diff
@@ -11,7 +11,7 @@ static char _sigma_clip_fast_docstring[] = "Compute sigma clipping";
 
 /* Declare the C functions here. */
 static void _sigma_clip_fast(
-    char **args, npy_intp *dimensions, npy_intp* steps, void* data);
+    char **args, npy_intp const *dimensions, npy_intp const* steps, void* data);
```

Comment:
for consistency
```suggestion
    char **args, npy_intp const* dimensions, npy_intp const* steps, void* data);
```

Reply:
Yeah, I'd recommend using clang-format. Didn't want to get into formatting issues.